app/views/projects/samples/index.html.erb

```diff
@@ -7,7 +7,7 @@
   <%= component.with_buttons do %>
     <div class="flex items-center space-x-2">
       <% if allowed_to?(:submit_workflow?, @project) %>
-        <%= link_to pipeline_selection_workflow_executions_submissions_path, data: { turbo_stream: true, controller: "action-link", action_link_required_value: 1 },
+        <%= link_to pipeline_selection_workflow_executions_submissions_path(:namespace_id => @project.namespace.id), data: { turbo_stream: true, controller: "action-link", action_link_required_value: 1 },
```

Comment:
Can you change the `:namespace_id => @project.namespace.id` to `namespace_id: @project.namespace.id`?